src/dm_logger.vala

```diff
@@ -97,10 +98,13 @@ namespace DMLogger
         string[] tokens = ( (!)line ).split( "\x01" );
         if ( print_verbose == true )
         {
-          stdout.printf( "Adding %lld = %s to mdb...\n", int64.parse( tokens[ 1 ] ), tokens[ 0 ] );
+          stdout.printf( "Adding %s %lld = %s to mdb...\n", tokens[ 0 ], int64.parse( tokens[ 2 ] ), tokens[ 1 ] );
         }
-        mdb.insert( int64.parse(tokens[ 1 ] ), tokens[ 0 ] );
+        
+        mdb_mini.insert( int64.parse(tokens[ 2 ] ), tokens[ 1 ] );
+        mdb.insert( tokens[ 0 ], mdb_mini );
```

Comment:
Das funktioniert so nicht...
Du musst zuerst nachsehen ob im mdb-Hash bereits ein Hash zur Komponente in `tokens[ 0 ]` gespeichert ist.
Wenn ja, dann musst du diesen laden und in diesen den Eintrag tokens[1] -> tokens[2] hinzufügen.
Wenn nein, dann musst du einen neuen Hash erstellen, den Eintrag tokens[1] -> tokens[2] hinzufügen und den neu erstellten Hash zu `mdb` hinzufügen (Key ist dann `tokens[ 0 ]`).